recipes/libarchive/all/conanfile.py

```diff
@@ -0,0 +1,188 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class LibarchiveConan(ConanFile):
+    name = "libarchive"
+    description = "C library for encoding, decoding and manipulating JSON data"
+    topics = ("conan", "libarchive", "tar", "data-compressor", "file-compression")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://libarchive.org"
+    license = "https://raw.githubusercontent.com/libarchive/libarchive/master/COPYING"
```

Comment:
```suggestion
    license = "BSD"
```
libarchive's license is a Frankstein, Apache2 + BSD 2 + ... But I vote for BSD, as it's described on its [wiki](https://github.com/libarchive/libarchive/wiki#what-is-libarchive)

Reply:
In other multi-licensing Conan recipes, we provided such link to a license file. Debian (it is usually a good example of how to deal with licenses) specifies multiple licenses too. In my opinion, we should not lose the information.

Reply:
I see, is just in case in the future we use the license field to populate Bintray page. Actually we have a [hook](https://github.com/conan-io/hooks/blob/master/hooks/bintray_updater.py) for it, but it not assigned as official.

Reply:
By the way, if the package directory `licences/` is more significant for licensing information than the license field in the conanfile.py, then letting license = "BSD" for information purposes is more than okay.

Reply:
far enough